test/FieldsEnumTests.cpp

```diff
@@ -0,0 +1,30 @@
+#include "FieldsEnum.hpp"
+#include <gtest/gtest.h>
+
+class FieldsEnumTests :public ::testing::Test
+{
+public:
+    FieldsEnum field;
+};
+
+TEST_F(FieldsEnumTests, assertCorrectWhiteField)
+{
+    field = FieldsEnum::WHITE;
+    ASSERT_EQ(field, FieldsEnum::WHITE);
```

Comment:
Such tests are obvious and are always true - it's language guarantee that assignment should always work.
We do not check if a == 3 after assignment a = 3;
Dedicated tests for enum are not needed. It will be used in another tests.